src/server/arenaApp.ts

```diff
@@ -1,9 +1,19 @@
 import { Express, RequestHandler } from 'express'
-
+import { User as _User } from '@openforis/arena-core'
 /**
  * Express app wrapper.
  */
 export interface ArenaApp {
   express: Express
   session: RequestHandler
 }
+
+declare global {
+  // eslint-disable-next-line @typescript-eslint/no-namespace
+  namespace Express {
+    // Express.User is defined as {}
+    // We extend it globally to be arena-core/User
+    // @ts-ignore
```

Comment:
why ts-ignore ?

Reply:
As in comment:
`     // Express.User is defined as {} `
```
Duplicate identifier 'User'.ts(2300)
index.d.ts(20, 19): 'User' was also declared here
```